src/libs/state-history.js

```diff
@@ -0,0 +1,8 @@
+import _ from 'lodash'
+
+
+export const set = newState => { window.history.replaceState(newState, '') }
+
+export const get = () => window.history.state
```

Comment:
TOL: If this abstraction layer is assuming that state will be an object, it might be nice if this did `window.history.state || {}` so the consumers don't have to